blackbox/docs/admin/privileges.rst

```diff
@@ -76,28 +76,40 @@ Hierarchical Inheritance of Privileges
     CREATE OK, 1 row affected (... sec)
 
     cr> create table if not exists doc.accounting (
-    ...   first_column integer primary key,
+    ...   id integer primary key,
     ...   second_column string,
     ...   third_column timestamp,
     ...   fourth_column object(strict) as (
     ...     key string,
     ...     value string
     ...   )
-    ... ) clustered by (first_column) into 5 shards;
+    ... ) clustered by (id) into 5 shards;
     CREATE OK, 1 row affected (... sec)
 
+    cr> INSERT INTO doc.accounting
+    ...   (id, second_column, third_column, fourth_column)
+    ...   VALUES (1, 'bla', 0, {key='blub', value='bluba'});
```

Comment:
maybe use some more meaningful fake data for the documentation.
don't like that we have bogus all over the place, but I think whenever we create new examples or touch existing, we should try to improve them